syntax = "proto3";

package meeting.health;

service HealthService {
    rpc Check (HealthCheckRequest) returns (HealthCheckResponse) {}
}

message HealthCheckRequest {
    string source = 1;  // 请求发起者标识，用于日志
}

message HealthCheckResponse {
    string status = 1;  // e.g. "SERVING"
    int64 timestamp = 2;  // 服务端响应时间（秒）
}